src/lib/dbLocalStorage.js

```diff
@@ -50,6 +50,13 @@ const removeUserProp = (db, userId, key) => {
     return setUser(db, userId, { ...other });
 };
 
+const getSelectWhereList = (db, key, value) => {
```

Comment:
this name sounds confusing, maybe call this getFilteredUsers to go on the same line that the existing getUsers